oem/ibm/pfw-sms-utils/pfw_sms_menu.hpp

```diff
@@ -0,0 +1,75 @@
+#ifndef PFW_SMS_MENU_IBM_H
+#define PFW_SMS_MENU_IBM_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <security/pam_appl.h>
+#include <stdbool.h>
+#include <string.h>
+
+    enum changePasswordReasonCode
+    {
+        PasswordChangeSuccessful = 0,
```

Comment:
can you please use enum name as per other pldm supported enum?
[https://github.com/ibm-openbmc/libpldm/blob/efb400697eede42f14ad6290660a4aecbc314141/include/libpldm/base.h#L20](url)

Reply:
Yes, I will do that.